src/lib/operators/join_hash.cpp

```diff
@@ -528,10 +553,7 @@ class JoinHash::JoinHashImpl : public AbstractJoinOperatorImpl {
       }
     } else {
       _copy_table_metadata(_left_in_table, _output_table);
-
-      if (_mode != JoinMode::Semi && _mode != JoinMode::Anti) {
-        _copy_table_metadata(_right_in_table, _output_table);
-      }
+      _copy_table_metadata(_right_in_table, _output_table);
```

Comment:
Why do we copy the metadata even for semi/anti? I believe the old version was correct.

Reply:
The old version copied for anything *but* semi/anti. Now that Semi and Anti are both always swapped, we do not need the check here, though.

Reply:
I missed that comment. Maybe this would be a good place to remember people that those are always swapped.

Reply:
added two comments as a reminder